libs/application/templates/marriage-conditions/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,55 @@
+import * as z from 'zod'
+import { parsePhoneNumberFromString } from 'libphonenumber-js'
+import * as kennitala from 'kennitala'
+import { YES, NO, MarriageTermination } from './constants'
+
+const emailRegex = /^[\w!#$%&'*+/=?`{|}~^-]+(?:\.[\w!#$%&'*+/=?`{|}~^-]+)*@(?:[A-Z0-9-]+\.)+[A-Z]{2,6}$/i
+const isValidEmail = (value: string) => emailRegex.test(value)
```

Comment:
There is `zod.string().email()` I believe.
Not a review comment since there may be cases where `refine` is preferred.